src/NativeRNDeviceInfoAndroid.ts

```diff
@@ -0,0 +1,154 @@
+import type { TurboModule } from 'react-native';
+import { TurboModuleRegistry } from 'react-native';
+
+export type LocationProviderInfo = {
+  [key: string]: boolean;
```

Comment:
Is it not possible to type it anyhow?

Reply:
https://github.com/react-native-device-info/react-native-device-info/blob/98de4817c8911349023af5725f0a789dabb8641c/src/internal/types.ts#L12-L14 Existing type